### DISCUSSION TITLE
Any way to set a maximum memory limit on inference mode ?

### DISCUSSION THREAD

**Discussion by Mactarvish** [facebookresearch/detectron2#3994] [Q&A]
I'm trying to running inference model on a K8S env, and I'm willing to set the maximum GPU memory limit to 8G, i.e. the rest part more than 8G shall be handled by CPU. Any way to set that? Thanks.

**Comment by ExtReMLapin** [comment]
Why do you need to ? 

What are you doing exactly ?
